cfme/roles.py

```diff
@@ -279,3 +279,179 @@ def _remove_from_all(roles, r_page):
         'about'
     ]
 }
+
+# Matches structure/string format of VerticalNavigation output for tree, not UI access control tree
+# TODO include non-vertical nav RBAC to settings, help
+# TODO RBAC goes deeper than veritcal nav, into accordions. example cloud intel -> Reports
```

Comment:
This should probably be in a yaml file, I'd prefer to leave that change to a follow-up PR.